src/config/config.js

```diff
@@ -1,9 +1,6 @@
-const dotenv = require('dotenv');
-const path = require('path');
+require('dotenv').config();
 const Joi = require('joi');
 
-dotenv.config({ path: path.join(__dirname, '../../.env') });
```

Comment:
Let's keep this line as it gives the flexibility to choose a `.env` file with a different name